src/components/multi-action-button/multi-action-button.test.js

```diff
@@ -181,290 +181,308 @@ context("Tests for MultiActionButton component", () => {
             .and("be.visible");
         });
     });
-  });
 
-  describe("pressing ArrowUp while MultiActionButton is open", () => {
-    it("should move focus to previous child button and should not loop to last button when first is focused", () => {
-      CypressMountWithProviders(
-        <>
-          <MultiActionButtonList />
-          <MultiActionButtonList />
-        </>
+    it("when customWidth is passed, should render Button with justify-content: space-between", () => {
+      CypressMountWithProviders(<MultiActionButtonList customWidth="70%" />);
```

Comment:
this test also needs `width` instead of `customWidth` (in the test description as well as the actual prop)